ui/perf.html

```diff
@@ -72,14 +72,18 @@
   <script src="js/models/job.js"></script>
   <script src="js/models/runnable_job.js"></script>
   <script src="js/models/resultset.js"></script>
+  <script src="js/models/user.js"></script>
   <script src="js/perf.js"></script>
   <script src="js/controllers/perf/compare.js"></script>
   <script src="js/controllers/perf/graphs.js"></script>
   <script src="js/controllers/perf/alerts.js"></script>
+  <script src="js/directives/treeherder/persona.js"></script>
   <script src="js/directives/perf/compare.js"></script>
   <script src="js/perfapp.js"></script>
   <script src="js/filters.js"></script>
   <!-- endbuild -->
 
+  <script src="https://login.persona.org/include.js"></script>
```

Comment:
I agree.  Just if persona.org is down for a couple hours the trees will
probably be closed.

On Wed, Nov 25, 2015 at 12:42 PM, William Lachance <notifications@github.com
> wrote:

> In ui/perf.html
> <https://github.com/mozilla/treeherder/pull/1159#discussion_r45916970>:
>
> >    <script src="js/directives/perf/compare.js"></script>
> >    <script src="js/perfapp.js"></script>
> >    <script src="js/filters.js"></script>
> >    <!-- endbuild -->
> >
> > +  <script src="https://login.persona.org/include.js"></script>
>
> Presumably this is just so we can take advantage of fixes to persona
> without having to update treeherder.
>
> If we can't trust persona.org, we're in a lot of trouble. :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mozilla/treeherder/pull/1159/files#r45916970>.
>